File: JavaScript/Very Easy/FindThePerimeterOfARectangle.js
/*
Find the Perimeter of a Rectangle
Create a function that takes length and width and finds the perimeter of a rectangle.

Examples
findPerimeter(6, 7) ➞ 26

findPerimeter(20, 10) ➞ 60

findPerimeter(2, 9) ➞ 22
*/

const findPerimeter = (length, width) => {
    return 2 * (length + width)
}

console.log(findPerimeter(6, 7))  //Output: 26

console.log(findPerimeter(20, 10)) //Output: 60

console.log(findPerimeter(2, 9)) //Output: 22

File: Java/VeryEasy/FindThePerimeterOfARectangle.java
/*
Find the Perimeter of a Rectangle

Create a function that takes length and width and finds the perimeter of a rectangle.

Examples:

findPerimeter(6, 7) ➞ 26
 */


public class FindThePerimeterOfARectangle {
    public static void main(String[] args) {
        int findPerimeter = parameters(6, 7);
        System.out.println(findPerimeter); //Output: 26
    }

    static int parameters(int length, int width) {
        int result = (length + width) * 2;
        return result;
    }
}
